### DISCUSSION TITLE
Can not log in with saved username.

### DISCUSSION THREAD

**Discussion by TommyBuho** [btcpayserver/btcpayserver#5522] [Technical Support]
Hi all. This is all local on my Umbrel with RazPi 4. 
I logged into BTCPay Server for the first time locally from my Umbrel server. I allowed my username and password to be stored on my keychain.  I was playing with it for several days setting things up without logging out. My computer logged me out automatically. When I try to log back in it says "the username is not in an email format". When I try all variations I can not log in.  So somehow I was logged in for the first time and my password manager saved the wrong username.  This is the first and ONLY username/login associated with BTCPS. 
I do have some dust bitcoin in the wallets so I do not want to just erase and start over. Is there a way to find the username/password sequence in the local login files?
Or is there a way to change the username and password. 

I do have the Recovery Seed. 

Thanks.

**Comment by dennisreimann** [comment]
Usually you don't use a username but email address to log in. See if you can find that instead of a username.

Other than that you have the option to reset the admin via this workaround: https://docs.btcpayserver.org/FAQ/ServerSettings/#forgot-btcpay-admin-password

**Reply by TommyBuho** [reply]
I am on an Umbrel . I have SSH's into it and can not find the directory with BTCpayserver-docker . 

> cd $BTCPAY_BASE_DIRECTORY/btcpayserver-docker/

Can you point me in the right direction?

**Reply by dennisreimann** [reply]
I am not sure about Umbrel and whether or not they have those scripts. From what I remember you might be able to at least access the BTCPay Server container this way: `scripts/app compose btcpay-server exec bash` and then see if the script is present. Otherwise please consult Umbrel about how to reset the account.

**Reply by TommyBuho** [reply]
Thanks for your responses. I have not been successful. Umbrel is great when I do not need help.

**Reply by dennisreimann** [reply]
Sorry, I don't have an Umbrel instance to test with. I guess it's best to bring this up as an [issue on their repo](https://github.com/getumbrel/umbrel/issues) or join their support channels to see if someone else can guide you.

**Reply by TommyBuho** [reply]
Is there a place where I can check the log files to see what email I used to login with?

**Reply by dennisreimann** [reply]
You can run `docker ps` to get the name of the BTCPay Server container and then use `docker logs CONTAINER_NAME`.

**Reply by TommyBuho** [reply]
I did find as you suggested docker ps and docker logs CONTAINER_NAME above but now a bit lost. 

umbrel@umbrel:~/umbrel/app-data/btcpay-server $ docker logs btcpay-server_web_1
info: Configuration:  Data Directory: /data
info: Configuration:  Configuration File: /data/Main/settings.config
info: BTCPayServer.Plugins.PluginManager: Loading plugins from /data/plugins
info: BTCPayServer.Plugins.PluginManager: Adding and executing plugin BTCPayServer - 1.11.7
info: BTCPayServer.Plugins.PluginManager: Adding and executing plugin BTCPayServer.Plugins.Shopify - 1.11.7
info: BTCPayServer.Plugins.PluginManager: Adding and executing plugin BTCPayServer.Plugins.PointOfSale - 1.11.7
info: BTCPayServer.Plugins.PluginManager: Adding and executing plugin BTCPayServer.Plugins.PayButton - 1.11.7
info: BTCPayServer.Plugins.PluginManager: Adding and executing plugin BTCPayServer.Plugins.NFC - 1.11.7
info: BTCPayServer.Plugins.PluginManager: Adding and executing plugin BTCPayServer.Plugins.Crowdfund - 1.11.7
info: Configuration:  Supported chains: BTC
info: Configuration:  BTC: Explorer url is http://btcpay-server_nbxplorer_1:32838/
info: Configuration:  BTC: Cookie file is /data/.nbxplorer/Main/.cookie
info: Configuration:  Network: Mainnet
info: Configuration:  Root Path: /
info: Configuration:  Connection to NBXplorer's database successful, dashboard and reporting features activated.
info: PayServer:      Checking if any payment arrived on lightning while the server was offline...
info: PayServer:      Starting listening NBXplorer (BTC)
info: PayServer:      Start watching invoices
info: PayServer:      Starting payment request expiration watcher
info: PayServer:      0 pending payment requests being checked since last run
info: Configuration:  Now listening on: http://0.0.0.0:3003
info: PayServer:      Processing lightning payments...
info: PayServer:      BTC: Checking if any pending invoice got paid while offline...
info: PayServer:      BTC: 0 payments happened while offline
info: PayServer:      Connected to WebSocket of NBXplorer (BTC)
info: PayServer:      BTC (Lightning): Start listening https://10.21.21.9:8080/
info: PayServer:      BTC (Lightning): Payment detected via notification (5VFEDJ99oM3anU2U7UVdnX)
info: PayServer:      BTC (Lightning): No more invoice to listen on https://10.21.21.9:8080/, releasing the connection.
info: PayServer:      BTC (Lightning): Start listening https://10.21.21.9:8080/
info: PayServer:      BTC (Lightning): No more invoice to listen on https://10.21.21.9:8080/, releasing the connection.
warn: Configuration:  Should have sent email, but email settings are not configured
warn: Configuration:  Should have sent email, but email settings are not configured
warn: Configuration:  Should have sent email, but email settings are not configured
warn: Configuration:  Should have sent email, but email settings are not configured
